assets/js/uno.js

```diff
@@ -0,0 +1,281 @@
+////////////////////////
+// Card Property
+///////////////////////
+
+// Color
+var Red = 0

```

Comment:
variables good practice is always in `camelCase`, so it should start with a smallcap. Only use big cap for `contructor` variable